src/generateDapp.ts

```diff
@@ -56,6 +56,14 @@ export async function generateDapp(selection: Selections) {
       }
     };
 
+    const rename = async (dir: string, oldName: string, newName: string) => {
```

Comment:
Both functions should live in the `utils/helpers/` folder. Also, please add doc string for each function